### ISSUE TITLE
Review colour contrast of image Placeholder - UX

### ISSUE THREAD

**Issue by pjlee11** [bbc/simorgh#769]
Raised as an outcome of https://github.com/BBC-News/simorgh/pull/668

**Is your feature request related to a problem? Please describe.**
The colour contrast ratio of the placeholder image is lower than the accessibility guidelines of 4.5:1 - http://www.bbc.co.uk/guidelines/futuremedia/accessibility/mobile/design/colour-contrast. 

**Describe the solution you'd like**
UX should review the colour contrast and edit designs as necessary. Once updated designs are available code changes should be made. 

**Describe alternatives you've considered**
N/A

**Testing notes**
Check the colour contrast ratio of the new image placeholder. 

**Screenshots**
![image](https://user-images.githubusercontent.com/7791726/46545969-bec9b900-c8bf-11e8-9b97-7bbf086b10aa.png)

**Comment by pjlee11** [comment]
cc @greenc05

**Comment by pjlee11** [comment]
<img width="536" alt="image placeholder - 16x9" src="https://user-images.githubusercontent.com/7791726/46546712-cbe7a780-c8c1-11e8-8b0c-57df132f319f.png">

**Comment by greenc05** [comment]
Thanks @pjlee11. I agree with Natalie, this is a presentational image and therefore doesn't need to meet contrast ratios as it's not content. I'm not even sure why it says 'BBC'? The image might not be BBC copyright, it just says that for presentational effect, it's is not content.

**Comment by pjlee11** [comment]
@greenc05 thanks for confirming this, will close this issue and merge #668
